### ISSUE TITLE
IDAPINavigator

### ISSUE THREAD

**Issue by TobiasWantzen** [IDPF/epubcheck#623]
When checking EPUBs I get this error message:
»java[13440:1f07] Can't open input server /Library/InputManagers/IDAPINavigator«

I'm running MacOS X 10.7, Java8, epubcheck 4.0.0.

Tobias

**Comment by rdeltour** [comment]
This rings no bells. Are you using a JDK or a JRE? Can you please give the version details? (type `java -version` in the Terminal). Thx!

**Comment by TobiasWantzen** [comment]
Thanks for your quick reply!

java version "1.8.0_60"
Java(TM) SE Runtime Environment (build 1.8.0_60-b27)
Java HotSpot(TM) 64-Bit Server VM (build 25.60-b23, mixed mode)

**Comment by TobiasWantzen** [comment]
Currently there are problems using Kindle Previewer, which needs java6. epubcheck needs java7+. I've been testing for a while, but for now I can't get multiple java versions to run simultaneously.

Tobias

**Comment by rdeltour** [comment]
Thanks, I'll try to reproduce on a virtual machine. I've heard about the Kindle Previewer issue, but unfortunately there's nothing we can do. Note that EpubCheck can still run with Java 6, it's just that some (very specific) tests on images will not be available. 99.9% of the tests will be ok (it's not an accurate figure, but you get the gist).

**Comment by tofi86** [comment]
@TobiasWantzen there exist some tutorials how to hot-fix KindlePreviewer, e.g. https://ebook-world.eu/reader/kindle-previewer-wieder-ans-laufen-bringen-osx

**Comment by TobiasWantzen** [comment]
Thanks tofi86 for your hint. I already knew this one. Since I use Parallels on my Mac, I installed KindlePreviewer on my windows installation.

@rdeltour: 
Meanwhile I updated to 10.10.5, but the error message stays the same:
java[2530:30612] Can't open input server /Library/InputManagers/IDAPINavigator

Tobias

**Comment by TobiasWantzen** [comment]
After two days of searching and testing, I finally found something:
InputManagers are pre 10.7 system components. I bought my Mac with 10.7 preinstalled, but there is a chance, that it had 10.6 installed and updated before. I finally took my courage in both hands and simply deleted the complete folder **/library/InputManagers/** from my system ... and the error message is gone!
I don't know, if you'd like to close the topic or hold it for further investigations. For me it's solved.

Thanks!
Tobias
